src/transformers/training_args.py

```diff
@@ -207,18 +207,26 @@ class TrainingArguments:
             Random seed that will be set at the beginning of training. To ensure reproducibility across runs, use the
             :func:`~transformers.Trainer.model_init` function to instantiate the model if it has some randomly
             initialized parameters.
+        bf16 (:obj:`bool`, `optional`, defaults to :obj:`False`):
```

Comment:
Might be surprising as an argument name? Won't users expect `--bfloat16`?

Reply:
this just mimics `--fp16` style - how could the `--bfloat16` style fit in here? i.e. if the other argument were `--float16` then `--bfloat16` fits.

Reply:
Ok

Reply:
I prefer `--bfloat16`, but @stas00 is right, `--bf16` is consistent.